bindings/python/docs/Demonstration/new-orbit-constructors.ipynb

```diff
@@ -0,0 +1,7874 @@
+{
```

Comment:
Good for end-users, thanks for that 🙏 Let's keep it here for now but we'll probably move all demo notebook and tutorials to bae project in a combined cleaning effort